test/check-application

```diff
@@ -256,9 +256,10 @@ class TestFiles(testlib.MachineCase):
         b.wait_visible("[data-item='urandom']")
 
         # Non-existing directory
+        b.wait_visible("#dropdown-menu")
         b.go("/files#/?path=/doesnotexists")
         b.wait_in_text(".pf-v5-c-empty-state__body", "No such file or directory")
-        b.wait_visible("#dropdown-menu:disabled")
+        b.wait_not_present("#dropdown-menu")
```

Comment:
Uhhh, the sidebar no longer contains a dropdown. So why should it hide?

Reply:
well the whole header goes away in a non-existing directory. But maybe it is not important to test it?

![image](https://github.com/user-attachments/assets/7978b94d-5386-4842-8ece-71e2962fb875)

Reply:
Oooo, right it does. That should hmmm not happen? It makes sense for folder related actions, but not for global actions like showing keyboard shortcuts